ext/ldap/ldap.stub.php

```diff
@@ -2,131 +2,135 @@
 
 /** @generate-class-entries */
 
-/** @strict-properties */
-final class LDAP
-{
+namespace LDAP {
+    /** @strict-properties */
+    final class LDAP
```

Comment:
Can you please use LDAP\Connection? That's what is consistent with other similar classnames, and what Nikita wrote in https://externals.io/message/114070#114261